package.json

```diff
@@ -33,6 +33,12 @@
   },
   "homepage": "https://github.com/fedor/node-no-config#readme",
   "dependencies": {
-    "co": "^4.6.0"
+    "babel-core": "^6.3.15",
```

Comment:
@anuragCES, for new dependencies, since they are needed only for tests, better to specify them as *development* dependencies. To do this, use `npm install <PACKAGE> --seve-dev` (https://docs.npmjs.com/cli/install)